packages/babel-plugin-i18n/README.md

```diff
@@ -0,0 +1,18 @@
+i18n
```

Comment:
Re: Naming. I think the title here should reflect the fact that it's a Babel plugin. Maybe at least "i18n Babel Plugin".

But aside from it being scoped to WordPress, and WordPress using gettext for i18n, I feel a bit odd calling this a "i18n Babel Plugin", since gettext is just one method of internationalizing.

The equivalent PHP tool is called `makepot.php`, so maybe `babel-plugin-makepot` if we want consistency (whether or not the name on its own is great).

Otherwise some variations of words in `babel-plugin-extract-gettext-strings` ? `-extract-strings`? `-extract-gettext` ? `-gettext-strings` ?

Reply:
`babel-plugin-makepot` sounds good to me